src/utils.ts

```diff
@@ -26,19 +27,7 @@ export function parseHTTPHeadersQuotedKeyValueSet(
   if (!matches) throw new YError('E_MALFORMED_QUOTEDKEYVALUE', contents);
 
   const data = matches
-    .map((part, partPosition) => {
```

Comment:
I think importing a library to just do this is overkill. Something like `return [pair[0], pair.slice(1).join('')]` will fit the needs.